### ISSUE TITLE
Incorrect `warning[unused-ignore]` for valid rule name `prefer-timestamp-tz`

### ISSUE THREAD

**Issue by hc-12** [sbdchd/squawk#508]
The squawk linter incorrectly reports `warning[unused-ignore]: unknown name prefer-timestamp-tz` even though `prefer-timestamp-tz` is a valid rule that the same linter enforces elsewhere in the codebase.

Steps to Reproduce
- Create a migration file with a timestamp column
- Add a squawk-ignore comment for the prefer-timestamp-tz rule
- Run the squawk linter on the file

```sql
-- This file: example_migration.sql
create table test_table (
  -- squawk-ignore prefer-timestamp-tz
  created_at timestamp default current_timestamp,
  other_field text
);
```

Actual Behavior
The squawk linter reports:

```
warning[unused-ignore]: unknown name prefer-timestamp-tz
 --> example_migration.sql:3:20
  |
3 |   -- squawk-ignore prefer-timestamp-tz
  |                    -------------------
  |
```

Yet in the same output, it recognises `prefer-timestamp-tz` as a valid rule when warning about timestamp columns:

```
warning[prefer-timestamp-tz]: When Postgres stores a datetime in a `timestamp` field, Postgres drops the UTC offset. This means 2019-10-11 21:11:24+02 and 2019-10-11 21:11:24-06 will both be stored as 2019-10-11 21:11:24 in the database, even though they are eight hours apart in time.
 --> example_migration.sql:4:14
  |
4 |   created_at timestamp default current_timestamp,
  |              ---------
  |
  = help: Use timestamptz instead of timestamp for your column type.

Find detailed examples and solutions for each rule at https://squawkhq.com/docs/rules
Found 2 issues in 1 file (checked 1 source file)
```

Expected Behavior
The `-- squawk-ignore prefer-timestamp-tz` comment should suppress the `prefer-timestamp-tz` warning for the line following it, and no warning[unused-ignore] should be reported.

Squawk version: 2.9.0
